### PR TITLE
ethdb: add LDBDatabase Meter test to verify metrics

### PR DESCRIPTION
**Pull request by Bo-Ye**
ethdb: add LDBDatabase Meter test to verify metrics

### CODE REVIEW

File: ethdb/database_test.go [hunk 1]
```diff
@@ -49,15 +49,18 @@ var test_values = []string{"", "a", "1251", "\x00123\x00"}
 func TestLDB_PutGet(t *testing.T) {
 	db, remove := newTestLDB()
 	defer remove()
+	//enable metrics
+	metrics.Enabled = true
```
Comment (egonelbre) [inline ethdb/database_test.go]: This is racy because some tests that read the global variable can be running in parallel.

Comment (Bo-Ye) [inline ethdb/database_test.go]: good catch, I can fix that.

**Comment by Bo-Ye** [comment]
I was working on the Mac, I'm not sure why it failed on windows. Can I have a look at it on the windows to see if it is difficult to fix.

**Comment by Bo-Ye** [comment]
Just had a quick look at why the CI building failed. My guesswork is the CI server is too fast to report execution time of some tests is larger than zero. I can fix that.

**Pull request closed without merging**